modules/AOR_Reports/aor_utils.php

```diff
@@ -340,70 +302,70 @@ function getPeriodEndDate($dateTimePeriodListSelected)
             $datetimePeriod = new DateTime();
             break;
         case 'this_week':
-            $datetimePeriod = new DateTime("next week monday");
-            $datetimePeriod->setTime(0, 0, 0);
+            $datetimePeriod = new DateTime("this week sunday");
```

Comment:
Use single quotes over double where possible.
```suggestion
            $datetimePeriod = new DateTime('this week sunday');
```